### PR TITLE
chore(firestore-bigquery-export) updated example query to include deleted records

### PR DESCRIPTION
**Pull request by dackers86**
Updated document query for  https://github.com/firebase/extensions/issues/652

### CODE REVIEW

**Comment by dackers86** [comment]
@iislucas example updated!

**Comment by iislucas** [comment]
Thanks! 

Actually I think the new query will delete even the latest timestamped change, so that probably isn't good. Also, I think you should be subtracting time, not adding, right? Perhaps you want something more like this: 

```
DELETE FROM `TABLENAME_raw_changelog`
WHERE (document_name, timestamp) IN
(
WITH latest AS (
    SELECT MAX(timestamp) as timestamp, document_name
    FROM `TABLENAME_raw_changelog`
    GROUP BY document_name
)
SELECT (t.document_name, t.timestamp)
FROM `TABLENAME_raw_changelog` AS t
JOIN latest  ON (t.document_name = latest.document_name )
WHERE t.timestamp != latest.timestamp 
AND t.timestamp < TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 MONTH)
)
```

**Comment by dackers86** [comment]
> Thanks!
> 
> Actually I think the new query will delete even the latest timestamped change, so that probably isn't good. Also, I think you should be subtracting time, not adding, right? Perhaps you want something more like this:
> 
> ```
> DELETE FROM `TABLENAME_raw_changelog`
> WHERE (document_name, timestamp) IN
> (
> WITH latest AS (
>     SELECT MAX(timestamp) as timestamp, document_name
>     FROM `TABLENAME_raw_changelog`
>     GROUP BY document_name
> )
> SELECT (t.document_name, t.timestamp)
> FROM `TABLENAME_raw_changelog` AS t
> JOIN latest  ON (t.document_name = latest.document_name )
> WHERE t.timestamp != latest.timestamp 
> AND t.timestamp < TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 MONTH)
> )
> ```

Apologies, I understand now in terms retaining the latest and removing older records. 

That query looks great, however the last `WHERE` clause would be an issue....

```
AND t.timestamp < TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 MONTH)
```

` TIMESTAMP_SUB` does not support monthly intervals. As much as I would rather use timestamps, do you think the following would be a valid substitution?

```
AND DATETIME(t.timestamp) < DATE_ADD(CURRENT_DATETIME(), INTERVAL 1 MONTH)
```

**Comment by iislucas** [comment]
Small tweak: I would just make it `INTERVAL -1 MONTH` so that it's a date in the past instead of the future?

but otherwise using dates is fine, and DATE_ADD is good; sorry I didn't realise that TIMESTAMP_SUB didn't support month time intervals.

File: firestore-bigquery-export/guides/EXAMPLE_QUERIES.md [hunk 1]
```diff
@@ -116,7 +116,21 @@ If you want to clean up data from your `changelog` table, use the following
 
 ```sql
 /* The first WHERE clause is the start of the time range, and the second WHERE
-   clause is the end of the time range. The query below deletes every row in a
-   24-hour time frame on the 4th September, 2020. */
-DELETE FROM `[PROJECT ID].[DATASET ID].[CHANGELOG TABLE ID]` WHERE '2020-09-04 00:00:00' < timestamp AND '2020-09-05 00:00:00' > timestamp
+   clause is the end of the time range. The query below deletes any rows below that are over one
+   month old and have been deleted. */
```
Comment (Salakar) [inline firestore-bigquery-export/guides/EXAMPLE_QUERIES.md:120]:
```suggestion
   month old. */
```

File: firestore-bigquery-export/guides/EXAMPLE_QUERIES.md [hunk 2]
```diff
@@ -112,11 +112,23 @@ WHERE favorite_numbers_index = 0
...
+DELETE FROM `TABLENAME_raw_changelog`
+WHERE (document_name, timestamp) IN
+(
+  WITH latest AS (
+    SELECT MAX(timestamp) as timestamp, document_name
+    FROM `TABLENAME_raw_changelog`
+    GROUP BY document_name
+  )
+  SELECT (t.document_name, t.timestamp)
+  FROM `TABLENAME_raw_changelog` AS t
```
Comment (Salakar) [inline firestore-bigquery-export/guides/EXAMPLE_QUERIES.md:129]:
```suggestion
DELETE FROM `[PROJECT ID].[DATASET ID].[CHANGELOG TABLE ID]`
WHERE (document_name, timestamp) IN
(
  WITH latest AS (
    SELECT MAX(timestamp) as timestamp, document_name
    FROM `[PROJECT ID].[DATASET ID].[CHANGELOG TABLE ID]`
    GROUP BY document_name
  )
  SELECT (t.document_name, t.timestamp)
  FROM `[PROJECT ID].[DATASET ID].[CHANGELOG TABLE ID]` AS t
```

**Review by Salakar — APPROVED** [review]

**Comment by huangjeff5** [comment]
Can you take another look at this one? It fails the validate / formatting check.

**Comment by dackers86** [comment]
@huangjeff5 All checks now passing

**Pull request merged**